cfgrammar/src/lib/yacc/parser.rs

```diff
@@ -485,6 +481,7 @@ impl YaccParser {
             }
             i = j;
         }
+        let (i, _) = self.parse_to_eol(i)?;
```

Comment:
Can we squish this line and the below to `Ok(self.parse_ws(self.parse_to_eol(i).0, true)?)`?

Reply:
I was wondering about this myself, but hadn't checked since I hadn't seen the pattern used anywhere.  Apparently no (borrow checker, both take `&mut self`)

Reply:
Ah! Good point.